summary/percentiles.go

```diff
@@ -88,26 +91,31 @@ func (self *resource) Add(p info.Percentiles) {
 	}
 	self.mean.Add(p.Mean)
 	// Selecting 90p of 90p :(
+	// TODO(mvdan) what do we do with 50p and 95p?
 	self.samples = append(self.samples, p.Ninety)
 }
 
 // Add a single sample. Internally, we convert it to a fake percentile sample.
 func (self *resource) AddSample(val uint64) {
 	sample := info.Percentiles{
-		Present: true,
-		Mean:    val,
-		Max:     val,
-		Ninety:  val,
+		Present:    true,
+		Mean:       val,
+		Max:        val,
+		Fifty:      val,
+		Ninety:     val,
+		NinetyFive: val,
 	}
 	self.Add(sample)
 }
 
 // Get max, average, and 90p from existing samples.
-func (self *resource) GetPercentile() info.Percentiles {
```

Comment:
optional nit: Can we rename to GetAllPercentiles() instead. It makes searching/reading names easier if they don't differ in just the plural.